calebwagner/uteach-ilearn-server
# Cjw message viewset

#### Changes Made

1. Added file `messages.py` to `views` directory and modified the file to include CRUD functionality.
1. Modified file `__init__.py` to include import `MessageView` function.
1. Modified file `urls.py` to include url path the message viewset.
1. Modified file `models/message.py` to change attributes and add related names.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-message-viewset
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
- [ ] in Postman GET all messages with `http://localhost:8000/messages`
- [ ] perform a POST request with ...
`{
    "title": "Did you get my last message?",
    "user": 1,
    "recipient": 2,
    "timestamp": "http://localhost:8000/1629403224",
    "read": false,
    "description": "This is all you need to bring."
}`

- [ ] perform a GET request with `http://localhost:8000/messages/3`
- [ ] perform a DELETE request with `http://localhost:8000/messages/3`
